proto/prysm/v1alpha1/BUILD.bazel

```diff
@@ -263,8 +281,10 @@ ssz_gen_marshal(
     objs = [
         "BeaconBlocksByRangeRequest",
         "BlobSidecarsByRangeRequest",
+        "DataColumnSidecarsByRangeRequest",
```

Comment:
`DataColumnSidecarsByRangeRequest` and `MetaDataV2` are not defined anywhere. I think we mistakenly carried them over from the peer das branch

Reply:
Fixed in 3324b1def2683893a488eea42137b86ebb17efaf.